rte/mo_rte_lw.F90

```diff
@@ -225,6 +250,9 @@ function rte_lw(optical_props, top_at_1, &
                                sfc_emis_gpt, sources%sfc_source,       &
                                gpt_flux_up, gpt_flux_dn)
         !$acc exit data delete(optical_props%tau, optical_props%ssa, optical_props%g)
+        else
+           write(error_msg, "('lw_solver(...ty_optical_props_2str...) processing type canbe (0,1). entered: ', I2)") type2stream 
```

Comment:
This error checking isn't needed if we use a logical variables - but also please note that we never call `write()` within the core of `rte` but instead return strings.